CONTRIBUTING.md

```diff
@@ -285,14 +288,14 @@ a copy of the compiler rather than replacing it with `v self`.
 |                                   |                                                                                                                     |
 | `trace_gen`                       | Prints all the strings written to the generated C file. Very verbose.                                               |
 | `trace_cgen_stmt`                 | Prints details about the statements that are being processed by cgen.                                               |
-|                                   |        Use it for panics in cgen, to see the closest input V source line, that caused the panic.                    |
-|                                   |        Note: you need `v -no-parallel -d trace_cgen_stmt -o w cmd/v` to make sense of the output of that,           |
-|                                   |        otherwise by default cgen uses several threads, and the lines that are printed are out of order.             |
+|                                   | Use it for panics in cgen, to see the closest input V source line, that caused the panic.                           |
+|                                   | Note: you need `v -no-parallel -d trace_cgen_stmt -o w cmd/v` to make sense of the output of that,                  |
+|                                   | otherwise by default cgen uses several threads, and the lines that are printed are out of order.                    |
```

Comment:
The indent here was meaningful. It made clear that those lines belong to the same description, and are not for something else.

Reply:
The rendering here would be the same in the browser despite the indent. But I fully agree that it's good to keep when viewing the raw markdown!